lms/views/lti/basic_launch.py

```diff
@@ -274,6 +280,7 @@ def _configure_assignment(self, assignment):
             document_url=self.request.parsed_params["document_url"],
             group_set_id=self.request.parsed_params.get("group_set"),
             course=self.course,
+            auto_grading_config=self.request.parsed_parms.get("auto_grading_conig"),
```

Comment:
Here we'll take the new config of the assignment after editing and update the assignment.